tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -19,24 +19,28 @@ import (
 	"io/ioutil"
 	"math"
 	"net/http"
+	"os"
 	"path"
 	"strings"
 	"testing"
 
-	"istio.io/istio/pkg/test/framework/label"
+	"fortio.org/fortio/periodic"
```

Comment:
Can you avoid pulling in fortio? This seems superfluous.

Reply:
Needed fortio to be able to generate traffic...This library imports https://github.com/istio/istio/blob/c11ef9cc66c77eefc3e4feb7797df4de82e605c8/tests/integration/mixer/util.go fortio but is specific for Http, can add curl ability here?